src/main/java/com/chess/backend/repository/Firebase.java

```diff
@@ -44,8 +73,6 @@ public Firebase() {
         catch(IOException e) {
             System.out.println("ERROR: invalid service account credentials. See README.");
             System.out.println(e.getMessage());
-
-            System.exit(1);
```

Comment:
Please leave it like it is ;D
It makes no sense if we leave the server running with wrong service account credentials ;)